Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM kbase/kb_jre:latest as dockerize
 
-FROM python:3.6-alpine
+FROM python:3.9.19-alpine
```

Comment:
Does the readme need to be updated based on this change?

Reply:
yes, I will upgrade `README.md` as well as `RELEASE_NOTES.md` in a separate PR

Reply:
:+1: